DuckDuckGo/FileDownload/Model/DownloadListViewModel.swift

```diff
@@ -65,6 +67,33 @@ final class DownloadListViewModel {
         }
     }
 
+    private func setupErrorBannerBinding() {
+        $items.flatMap { items in
+                Publishers.MergeMany(items.map { $0.$state })
+            }.map { state in
+                if case .failed(let error) = state {
+                    if error.isNSFileReadUnknownError && self.isSpecificMacOSVersion() {
+                        return true
+                    }
+                }
+
+                return false
+            }
+            .removeDuplicates()
+            .receive(on: DispatchQueue.main)
+            .sink { [weak self] showError in
+                self?.shouldShowErrorBanner = showError
+            }
+            .store(in: &cancellables)
+    }
+
+    private func isSpecificMacOSVersion() -> Bool {
```

Comment:
How about renaming it to e.g. `isAffectedMacOSVersion`, and/or adding a link to Asana task in a comment?